models/calendar.yml

```diff
@@ -109,13 +109,21 @@
 
     - 8:00 PM:
         type: Talk
-        emoji: 🎉
-        title: TBD
-        description: TBD
+        emoji: 🚫☕️
```

Comment:
Let me know if you prefer the emoji to be a single emoji

Reply:
Multiple emoji is :+1::+1: